recipes/cnats/all/conanfile.py

```diff
@@ -85,21 +85,20 @@ def package(self):
         cmake = CMake(self)
         cmake.install()
         if self.settings.os == "Windows" and self.options.shared:
-            mkdir(self, os.path.join(self.package_folder, "bin"))
-            rename(self, os.path.join(self.package_folder, "lib", f"{self._nats_library_name}.dll"), os.path.join(self.package_folder, "bin", f"{self._nats_library_name}.dll"))
+            copy(self, pattern="*.dll", dst=os.path.join(self.package_folder, "bin"), src=os.path.join(self.package_folder, "lib"))
+            rm(self, "*.dll", os.path.join(self.package_folder, "lib"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
 
-    @property
-    def _nats_library_name(self):
-        suffix = "" if self.options.shared else "_static"
-        debug = "d" if self.settings.build_type == "Debug" else ""
-        return f"nats{suffix}{debug}"
-
     def package_info(self):
-        self.cpp_info.libs = [self._nats_library_name]
+        suffix = "" if self.options.shared else "_static"
+        lib_name = f"nats{suffix}"
+        self.cpp_info.libs = [lib_name]
```

Comment:
I think the lib_name is now wrong for Debug as it does not add the debug suffix when needed